libraries/rush-lib/src/logic/operations/OperationExecutionRecord.ts

```diff
@@ -207,18 +216,22 @@ export class OperationExecutionRecord implements IOperationRunnerContext {
       // normalizeNewlineTransform --1--> stderrLineTransform --2--> removeColorsTransform --> projectLogWritable
       //                                                        |
       //                                                        +--> stdioSummarizer
```

Comment:
This code is building up a complicated chain of transforms, that is not easy to understand from looking at the code.  That is why this `//` comment tries to illustrate how the transforms are connected, based on variable names chosen to be hopefully intuitive. The `//` comment tells a easy-to-understand story, something like:

*"First we normalize the newlines, then splitter `1` sends its data to the collated console output, unless quiet mode is enabled.  The other half of splitter `1` uses StderrLineTransform to ensure STDERR and STDOUT aren't mixed on the same line, the the second half of splitter `2` sends the result to `stdioSummarizer` for the report at the end of the build.  The first half of `2` removes colors and writes to the .log file, which makes sense because files shouldn't contain color codes."*

This PR is changing the topology; should we try to reflect that in the code comments?

Reply:
Updated the topology, should I also add a text comment here like you have or is the pipeline diagram sufficient?